### PR TITLE
Plugin cleanups

### PR DESCRIPTION
**Pull request by xerus2000**

### CODE REVIEW

File: plugin/src/test/sc/plugin2021/PieceTest.kt [hunk 1]
```diff
@@ -40,14 +42,22 @@ class PieceTest: WordSpec({
             }
         }
         "converted to String" Should {
```
Comment (anarchuser) [inline plugin/src/test/sc/plugin2021/PieceTest.kt:44]: If you really prefer `should` over `Should` then please be consistent

**Review by anarchuser — CHANGES_REQUESTED** [review]
Looks good generally, but I think you should do the translations to German separately. Cleaning up code and translating messages are two different things

File: plugin/src/test/sc/plugin2021/helper/MoveParser.kt [hunk 2]
```diff
@@ -35,29 +36,32 @@ object MoveParser {
...
-        }
-        listOf(  // Piece Shape using enum:
-                "BLUE|TRIO_I:0:0|3,18"   to SetMove(Piece(Color.BLUE, 3, Rotation.NONE, false, Coordinates(3, 18))),
-                "YELLOW|PENTO_I:1:1|0,2" to SetMove(Piece(Color.YELLOW, 14, Rotation.RIGHT, true, Coordinates(0, 2))),
-                "RED|PENTO_T:3:0|13,15"  to SetMove(Piece(Color.RED, 10, Rotation.LEFT, false, Coordinates(13, 15))),
-                "GREEN|skip" to SkipMove(Color.GREEN)
-        ).forEach{
-            parse(it.first).toString() shouldBe it.second.toString()
+    init {
+        runBlocking {
```
[the remark this answers is not in the document]
Comment (xerus2000) [reply plugin/src/test/sc/plugin2021/helper/MoveParser.kt:40]: yes, `forAll` is blocking

File: plugin/src/test/sc/plugin2021/PieceTest.kt (hunk 1 above)
Comment (xerus2000) [reply plugin/src/test/sc/plugin2021/PieceTest.kt:44]: hah right, missed that ^^

**Comment by xerus2000** [comment]
> Looks good generally, but I think you should do the translations to German separately. Cleaning up code and translating messages are two different things

They are separate, the translations are only one commit ;) https://github.com/CAU-Kiel-Tech-Inf/backend/pull/350/commits/e0e19422236fb21520be9f94436583549c07c70a

Sorry, I am too tired to make granular PRs, I just wanna get some stuff finished ^^

**Review by anarchuser — APPROVED** [review]
as long as you don't squash ^^

**Pull request merged**